sdks/python/apache_beam/testing/util.py

```diff
@@ -46,6 +56,26 @@ class BeamAssertException(Exception):
   pass
 
 
+# Used for reifying timestamps and windows for assert_that matchers.
```

Comment:
<!--thread_id:cc_151221908_t; commit:4fa4caa4b4991fe994ce3938f37be975421c6761; resolved:1-->
<!--section:context-quote-->
> **robertwb** wrote:
> Ideally one should be able to use any callable, e.g. hamcrest matchers, rather than have to implement a windowed variant of each. I misspoke about having a windowed_equals_to, we should have a `assert_that_windowed(pcoll, equal_to([WindowedValue(...), ...]))`, or `assert_that(pcoll, equal_to([WindowedValue(...), ...], reify_windows=True)`.

<!--section:body-->
Done.